src/lua/fiber.c

```diff
@@ -931,6 +951,8 @@ static const struct luaL_Reg fiberlib[] = {
 	{"parent_backtrace_enable", lbox_fiber_parent_backtrace_enable},
 	{"parent_backtrace_disable", lbox_fiber_parent_backtrace_disable},
 #endif /* ENABLE_BACKTRACE */
+	{"leak_backtrace_enable", lbox_fiber_leak_backtrace_enable},
+	{"leak_backtrace_disable", lbox_fiber_leak_backtrace_disable},
```

Comment:
Shouldn't we define these only if `ENABLE_LEAK_BACKTRACE` is set? Let's be consistent and define all the code dealing with leak backtraces (including the `fiber_leak_backtrace_enable` variable and the Lua API) under the macro.

Reply:
Done.